src/vaults/token/ShareToken.sol

```diff
@@ -42,7 +38,7 @@ contract ShareToken is ERC20, IShareToken {
     //----------------------------------------------------------------------------------------------
 
     /// @inheritdoc IShareToken
-    function file(bytes32 what, address data) external authOrHook {
+    function file(bytes32 what, address data) external auth {
```

Comment:
Fixes https://github.com/xmxanuel/centrifuge-protocol-v3-may-review/pull/14#discussion_r2079922517

Reply:
😲 Does this come from LP?